src/hooks/spoofing/write_player_game_state_data_node.cpp

```diff
@@ -1,13 +1,14 @@
 #include "hooking.hpp"
 #include <datanodes/player/CPlayerGameStateDataNode.hpp>
+#include "util/ped.hpp"
 
 namespace big
 {
 	bool hooks::write_player_game_state_data_node(rage::netObject* player, CPlayerGameStateDataNode* node)
 	{
 		auto ret = g_hooking->get_original<write_player_game_state_data_node>()(player, node);
 
-		if (g.spoofing.spoof_hide_god)
+		if (g.spoofing.spoof_hide_god && !is_player_in_cutscene && !is_player_in_interior(self::ped))
```

Comment:
`is_player_in_cutscene` is missing a method invocation, e.g. `is_player_in_cutscene(self::ped)`.

Moreover, it's awkward to call natives from this context. We could either reverse the natives and figure out what they're reading (not sure), or store these values in a global variable and have a loop update them from inside of a fiber.

Reply:
> is_player_in_cutscene is missing a method invocation, e.g. is_player_in_cutscene(self::ped).

same thing here, was trying to debug forgot to put back,  yes we can but I have no knowledge of how to do that

Reply:
leaving this open so others can see

Reply:
g_local_player->m_player_info->m_game_state == eGameState::InMPCutscene